src/components/Channel/__tests__/useIsMounted.test.js

```diff
@@ -1,4 +1,4 @@
-import { act, renderHook } from '@testing-library/react-hooks';
+import { act, renderHook } from '@testing-library/react';
```

Comment:
@testing-library/react-hooks does not seem to be continued - it has not been updated to be compatible with React v19.